packages/encoding/src/index.ts

```diff
@@ -18,6 +18,17 @@ export function uint8ArrayToHexString(input: Uint8Array): string {
   );
 }
 
+export const uint8ArrayFromHexString = (hexString: string): Uint8Array => {
```

Comment:
Do we have tests set up for this package yet? If so let's add tests for this new function (we have some that you can copy/paste from here: https://github.com/tkhq/frames/blob/5b5849ab288f0f1049c318288558d405a8533bc0/auth/index.test.js#L162-L181)